src/components/dashboard/ScanQR.web.js

```diff
@@ -38,7 +38,9 @@ const ScanQR = ({ screenProps }) => {
         if (!amount) {
           screenProps.push('Amount', { to: address, nextRoutes: ['Reason', 'SendQRSummary'] })
         } else {
-          screenProps.push('SendQRSummary', { to: address, amount, reason: 'From QR with Amount' })
+          // FIXME: nextRoute -> foo
+          // It was added as a workaround to the restrictions in the SendQRSummary.shouldNavigateToComponent
+          screenProps.push('SendQRSummary', { to: address, amount, reason: 'From QR with Amount', nextRoutes: ['foo'] })
```

Comment:
What's the purposes of nextRoutes?
Can you add a comment on ScanQRSummary.shouldNavigateToComponent what's the purpose of this?

Reply:
@sirpy, Ok. I've just fixed the workaround, and changed the condition in `SendQRSummary.shouldNavigateToComponent`.

Left a comment in the function expression (see: https://github.com/GoodDollar/GoodDAPP/pull/66/commits/b8f3c9078f7dee3dd672c7489c8f5d4cf271d49e)